captcha.module

```diff
@@ -334,17 +336,19 @@ function captcha_form_alter(array &$form, FormStateInterface $form_state, $form_
     ];
 
     if ($captcha_point !== NULL && $captcha_point->getCaptchaType()) {
-      $captcha_element['#title'] = t('CAPTCHA: challenge "@type" enabled', ['@type' => $captcha_point->getCaptchaType()]);
+      $markup = $captcha_point->status() ? '%type (<a href="@change">change</a>, <a href="@disable">disable</a>)' : '%type (<a href="@change">change</a>, <a href="@enable">enable</a>)';
+      $captcha_element['#title'] = $captcha_point->status() ? t('CAPTCHA: challenge "@type" enabled', ['@type' => $captcha_point->getCaptchaType()]) : t('CAPTCHA: challenge "@type" disabled', ['@type' => $captcha_point->getCaptchaType()]);
       $captcha_element['#description'] = t('Untrusted users will see a CAPTCHA here (<a href="@settings">general CAPTCHA settings</a>).',
         ['@settings' => Url::fromRoute('captcha_settings')->toString()]
       );
       $captcha_element['challenge'] = [
         '#type' => 'item',
         '#title' => t('Enabled challenge'),
-        '#markup' => t('%type (<a href="@change">change</a>, <a href="@disable">disable</a>)', [
+        '#markup' => t($markup, [
           '%type' => $captcha_point->getCaptchaType(),
-          '@change' => $captcha_point->url('edit-form', ['query' => drupal_get_destination()]),
-          //'@disable' => url("admin/config/people/captcha/captcha-points/$form_id/disable", array('query' => drupal_get_destination())),
+          '@change' => $captcha_point->url('edit-form', ['query' => Drupal::destination()->getAsArray()]),
+          '@disable' => $captcha_point->url('disable-form', ['query' => Drupal::destination()->getAsArray()]),
+          '@enable' => $captcha_point->url('enable-form', ['query' => Drupal::destination()->getAsArray()]),
         ]),
```

Comment:
disable was commented out, the link only pointed back to "home".

Since one test was testing the functionality, I thought I'd quickly reenable that link and add one to enable the captcha again (otherwise you would only be able to disable a captcha from the form directly, kind of weird, unless disable was renamed delete at some point).